DOCS/man/options.rst

```diff
@@ -3354,6 +3354,26 @@ Window
 
     ``never`` asks the window manager to never disable the compositor.
 
+``--x11-present=<no|auto|yes>``
+    Whether or not to use presentation statistics from xorg's presentation
+    extension (default: ``auto``).
+
+    mpv asks xorg for present events which it then may use for more accurate
+    frame presentation. This only has an effect if ``--video-sync=display-...``
+    is being used.
+
+    Drivers are detected using XRandr and depending on the conditions, are
+    added to a whitelist that enables the usage of presentation statistics
+    with the ``auto`` option. If the driver is detected to be amd/radeon,
+    intel, or nouveau (AKA the standard x86 Mesa drivers), then presentation
+    will be enabled. However, if a Nvidia driver is also detected at the same
+    time as a Mesa one, then it is removed from the whitelist and disabled.
+    Other drivers are not assumed to work, so they are not enabled by default.
```

Comment:
this reads very convoluted, I suggest something like:
"The `auto` option enumerates XRandr providers for autodetection. If any one of amd, radeon, intel or nouveau (the standard x86 Mesa drivers) are found and nvidia is NOT found, then presentation feedback is enabled. Other drivers are not assumed to work, so not enabled automatically."

Reply:
Thanks. I had trouble trying to word this right. That sounds better.